<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-V5MLE2I" failures="1" tests="5" name="Test" time="0.149" errors="0" timestamp="2021-12-28T09:13:46 IST">
  <testcase classname="testngdiscussion.SoftAssertDisc" name="testCase1" time="0.019">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at testngdiscussion.SoftAssertDisc.testCase1(SoftAssertDisc.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testCase1 -->
  <testcase classname="testngdiscussion.SoftAssertDisc" name="testCase2" time="0.004"/>
  <testcase classname="testngdiscussion.TestCasesForFailedExecution" name="home" time="0.001"/>
  <testcase classname="testngdiscussion.TestCasesForFailedExecution" name="loginTest" time="0.0">
    <skipped/>
  </testcase> <!-- loginTest -->
  <testcase classname="testngdiscussion.TestCasesForFailedExecution" name="a" time="0.007"/>
</testsuite> <!-- Test -->
